lang/ja.json

```
@@ -2028,18 +2028,18 @@
       "condition": "氷庭に到達する"
     },
     "ninja_town": {
-      "name": "Fake Ninja",
-      "description": "Scare me! Use ninjutsu!",
-      "condition": "After interacting with the unique stoplight with white lights, return to the ninja circle sub-area to witness their death"
+      "name": "ニセニンジャ",
+      "description": "恐れさせよ！忍術を使え！",
```

This line is from a popular catchphrase which doesn't mean anything.

> 吓我一跳！我释放忍术！

It literally means
```suggestion
      "description": "びっくりした！忍術を使え！",
```
But I think it can be better translated into whichever ninja-related neta you could think of.